apps/web/src/pages/templates/TemplatesListPage.tsx

```diff
@@ -154,33 +155,45 @@ function NotificationList() {
           />
         }
       />
+
       <TemplateListTableWrapper>
-        {hasTemplates ? (
-          <Table
-            onRowClick={onRowClick}
-            loading={isLoading || areNotificationGroupLoading}
-            data-test-id="notifications-template"
-            columns={columns}
-            data={templates}
-            pagination={{
-              pageSize: pageSize,
-              current: page,
-              total: totalTemplatesCount,
-              onPageChange: handleTableChange,
-            }}
-          />
-        ) : (
-          <TemplatesListNoData
-            readonly={readonly}
-            blueprints={popular?.blueprints}
-            isLoading={areBlueprintsLoading}
-            isCreating={isCreatingTemplateFromBlueprint}
-            allTemplatesDisabled={areBlueprintsLoading || !hasGroups}
-            onBlankWorkflowClick={() => handleRedirectToCreateTemplate(false)}
-            onTemplateClick={handleOnBlueprintClick}
-            onAllTemplatesClick={openModal}
-          />
-        )}
+        <LoadingOverlay
```

Comment:
We have a LoadingOverlay in the design system, if it helps..

Reply:
Good idea i will use it.